bento/src/androidTest/java/com/yelp/android/bento/test/CarouselComponentViewHolderTest.kt

```diff
@@ -142,6 +142,28 @@ class CarouselComponentViewHolderTest: ComponentViewHolderTestCase<Unit?, Carous
         }
     }
 
+    @Test
+    fun scrollCarousel_withItemVisibilityListener_getsCalledForEveryItem(){
+        val group = ComponentGroup()
+        group.addAll((1..20).map { SimpleComponent<Unit>(TestComponentViewHolder::class.java) })
+        var visibleCount = 0
+        group.registerItemVisibilityListener { index, isVisible ->
+            if(isVisible) visibleCount++
```

Comment:
nit: add space after `if`

Reply:
Oh, I'm too use to auto-formatting hooks :-D Will do!